Makefile

```diff
@@ -91,6 +91,11 @@ help:
 	@echo "        Create an RPM repository with modules"
 	@echo "    fixtures/rpm-with-modules-modified"
 	@echo "        Create an RPM repository with modules but without some dependencies."
+	@echo "    fixtures/rpm-modular"
+	@echo "        Create an RPM repository with modules, module defaults and module obsoletes."
+	@echo "        Packages taken from dnf-ci-fedora-modular repository and combined with "
+	@echo "        few modular metadata from dnf-ci-thirdparty-modular and dnf-ci-obsoletes"
+	@echo "        as those metadata is intended to operate above those packages."
```

Comment:
Could you add more detail about what modular metadata was copied?  Or was it all of it